oracle/README.md

```diff
@@ -2,17 +2,17 @@
 
 ## Overview
 
-Get metrics from oracle service in real time to:
+Get metrics from Oracle Database servers in real time to:
 
-* Visualize and monitor oracle states
-* Be notified about oracle failovers and events.
+* Visualize and monitor Oracle Database service status.
+* Be notified about Oracle Database cluster failovers and events.
```

Comment:
@marcweisman @l0k0ms @truthbk @handigarde Here we say this integration collects events, but on line 62 we say no events. Which is it?

Reply:
This is indeed wrong, we don't send events, and as far as I know we don't detect failovers - we just submit a service check.